### PR TITLE
Manual Cert Validation

### PR DESCRIPTION
**Pull request by 0xEmma**
## Types of changes

What types of changes does your code introduce?
*Put an `x` in the boxes that apply.*

- [ ] Bugfix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected).
- [ ] Documentation Update (if none of the other choices applies).

## Proposed changes
Add manual certification validation

## Checklist

*Put an `x` in the boxes that apply.*

- [x] I have read and followed the [CONTRIBUTING.md](https://github.com/hackthebox/Hackster/blob/main/CONTRIBUTING.md)
  doc.
- [x] Lint and unit tests pass locally with my changes.
- [x] I have added the necessary documentation (if appropriate).

## Additional context

Add any other context or screenshots here.

### CODE REVIEW

File: src/helpers/verification.py [hunk 1]
```diff
@@ -74,6 +74,28 @@ async def _check_for_ban(uid: str) -> Optional[Dict]:
 
     return ban_details
 
+async def process_certification(certid: str, name: str):
+    cert_api_url = f"{settings.API_V4_URL}certificate/lookup"
```
Comment (dimoschi) [inline src/helpers/verification.py:78]: There is a `/` missing

```suggestion
    cert_api_url = f"{settings.API_V4_URL}/certificate/lookup"
```

**Review by dimoschi — CHANGES_REQUESTED** [review]
Looks good, a minor fix is needed.

**Comment by 0xEmma** [comment]
Resolved @dimoschi

**Review by dimoschi — APPROVED** [review]
LGTM

**Review by dimoschi — APPROVED** [review]

**Pull request merged**